tests/social_buttons.js

```diff
@@ -0,0 +1,54 @@
+import { Selector, t, ClientFunction } from "testcafe";
+
+import { baseUrl, loginButton, validUserName, validPassword,
+        userNameInput, passwordInput } from "./login";
+const twitterButton = Selector('.social_twitter');
+const facebookButton = Selector('.social_facebook ');
+const linkedinButton = Selector('.social_linkedin');
+const getURL = ClientFunction(() => window.location.href);
+
+
+fixture`Social Buttons`
+    .page(baseUrl)
+
+
+test(`Verify social buttons - Twitter`, async t => {
+    await t
+        .setTestSpeed(0.8)
+        .typeText(userNameInput, validUserName)
+        .typeText(passwordInput, validPassword)
+        .click(loginButton)
+        .click(twitterButton)
+
+    const twitterPage = await getURL();
+
+    await t.expect(twitterPage).eql('https://twitter.com/saucelabs');
+});
+
+
+test(`Verify social buttons - Facebook`, async t => {
+    await t
+        .setTestSpeed(0.8)
+        .typeText(userNameInput, validUserName)
+        .typeText(passwordInput, validPassword)
+        .click(loginButton)
+        .click(facebookButton)
+
+    const facebookPage = await getURL();
+
+    await t.expect(facebookPage).eql('https://www.facebook.com/saucelabs');
+});
+
+
+test(`Verify social buttons - Linkedin`, async t => {
+    await t
+        .setTestSpeed(0.8)
+        .typeText(userNameInput, validUserName)
+        .typeText(passwordInput, validPassword)
+        .click(loginButton)
+        .click(linkedinButton)
+
+    const linkedinPage = await getURL();
+
+    await t.expect(linkedinPage).contains('https://www.linkedin.com/company/sauce-labs');
```

Comment:
Because page opens with the following link "https://www.linkedin.com/authwall?trk=bf&trkInfo=AQF8hV1abTHP-wAAAYZBvbhwPun7MxxKt6vyhKxi7ETD91aiQ3vsqT3yWj2H-epOEcl6IO8NNfNqE0SIfvr0QrA9Z14C1pwo_FDii8g1p2iM_8uPxY-fcxI1misfxtjruMFSJ7U=&original_referer=&sessionRedirect=https%3A%2F%2Fwww.linkedin.com%2Fcompany%2Fsauce-labs%2F" 
And making it .eql to it was failing the test. As the expected was the Linkedin Sauce Labs page, which it will not open into if the user is not logged in Linkedin. So, Catching the part of the Linkedin url partially, I thought it was sufficient